x-pack/plugins/cloud_security_posture/public/pages/rules/use_csp_rules.ts

```diff
@@ -27,6 +27,7 @@ export const useFindCspRuleTemplates = (
     () => {
       return http.get<GetCspRuleTemplateResponse>(FIND_CSP_RULE_TEMPLATE_ROUTE_PATH, {
         query: { packagePolicyId, page, perPage },
+        version: '1',
```

Comment:
Please double check that this change makes sense, for any `.versioned` endpoints in cloud_security_posture we should be passing a version (see my other comment).

Reply:
hey @jloleysens. for now this is the only endpoint in cloud_security_posture which using `.versioned`